src/main/java/org/zalando/pazuzu/feature/FeatureMetaDto.java

```diff
@@ -0,0 +1,107 @@
+package org.zalando.pazuzu.feature;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FeatureMetaDto {
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("description")
+    private String description;
+    @JsonProperty("author")
+    private String author;
+    @JsonProperty("updated_at")
+    private ZonedDateTime updatedAt;
+    @JsonProperty("dependencies")
+    private List<String> dependencies = new ArrayList<>();
+
+    public static FeatureMetaDto of(Feature feature) {
```

Comment:
Nope, circular dependencies are prevented during the creation and the update of features